src/definitions.rs

```diff
@@ -50,7 +50,7 @@ pub enum Action {
 
 impl State {
     #[inline]
-    pub fn entry_action(&self) -> Action {
+    pub fn entry_action(self) -> Action {
```

Comment:
Why are we changing `&self` to `self`?

Reply:
Because it implements `Copy` and would cause a clippy error otherwise.

Reply:
Could you also solve it here https://github.com/alacritty/vte/blob/7399615d1f94676b99defff0ebe9a385e26d7199/utf8parse/src/types.rs#L61

Reply:
Of course, looks like my clippy results were still cached.

Added clippy errors to all those libs, so CI should now catch that too.